platform-sdk/docs/proposals/proposal-process.md

```diff
@@ -0,0 +1,119 @@
+# Platform Design Proposal Process
+
+This document describes the process for creating and reviewing platform design proposals.
+
+The status of all active design proposals can be viewed
+at [Platform Design Proposals](https://github.com/orgs/hashgraph/projects/73/views/1) project board.
+
+---
+
+## Design Proposal Process Flow
+
+![](designProposalFlow.svg)
+
+## Creating A Proposal
+
+All proposals have an owner who is responsible for taking it through the process to completion. Proposals must be
+presented in the form of a PR containing the proposal in a subdirectory of `platform-sdk/docs/proposals/`. Proposals may
+contain diagrams and supporting materials. Diagrams must be in SVG format and all content must be in an editable
```

Comment:
What is the main file of a proposal? If we call it README.md it will be rendered in GitHub directly when you click on the folder. Might be a benefit.

Reply:
ooo, i like this a lot.

Reply:
I'm not personally a fan of this.

When navigating files in the IDE, it's really convenient to have unique names that you can just type in, and get to where you want to go. If everything is called `README`, then the only entry point is the directory structure

Having a file that isn't called `README` doesn't prevent github from rendering the doc, you just have to click the file.

Reply:
The folder name will still have the name of the proposal. That will show up in a search

Reply:
Of course it's still possible to search, but I can't type `command-o-<filename>` to jump right to it. Calling everything `README` makes the developer workflow harder, for what strikes me as minimal benefit.

Reply:
ok. i will revert this change

Reply:
Can i leave this process document name `README.md` so that is renders nicely in GitHub? Or do you prefer i give is a meaningful name?

Reply:
If github can show md files without needing to be README, then I agree with Austin.    This link should take you to the `rich diff` of a markdown file.   I am guessing it is based off of the last commit? https://github.com/hashgraph/hedera-services/pull/13747/files?short_path=e27a53e#diff-e27a53ec91c0f9e5fc505f6c74171f9e9b758963a734bfc0e725433f1b930b05

Reply:
The question is not about the PR view, but when you navigate to the repo's current version